content/user/libraries/license/Library-licenses.md

```diff
@@ -0,0 +1,26 @@
+<!--
+title: "Library Licenses"
+description: "View library licenses in the UI"
+tags: "user library licenses vulnerabilities sca"
+-->
+
+
+<!-- ## License Libraries 
+How do you provision licenses if you've paid for them? Like normal licenses in Org Settings? -->
+
```

Comment:
See comment above

Reply:
Looks good Kaitlyn. Just one comment on the title here. Ran it past Pauline too.
Instead of 'View Library Licenses' as the title can we say 'View Open Source Library Licenses'
(reason... some will appear as 'unknown', if they don't exists on our source list)

Reply:
& then the title of the section would be '**Open Source Licenses**' in the Docs menu item.
OS Licenses (if tight for space)

Reply:
The updates are in the PR: https://bitbucket.org/contrastsecurity/open-docs/pull-requests/392/contrast-34338-library-licensing/diff